install-dev/classes/controllerConsole.php

```diff
@@ -153,10 +153,13 @@ public function printErrors()
             if (!is_array($errors)) {
                 $errors = array($errors);
             }
-            echo 'Errors :'."\n";
+            echo 'Errors :'. PHP_EOL;
             foreach ($errors as $error_process) {
+                if (!is_array($error_process)) {
```

Comment:
@Quetzacoalt91 Having some errors here, don't know why $error_process isn't an array.

Reply:
I guess the array we merged does not always respect this rule.